handler.lua

```diff
@@ -41,6 +41,12 @@ function JwtClaimsValidateHandler:access(conf)
   JwtClaimsValidateHandler.super.access(self)
   local continue_on_error = conf.continue_on_error
 
+  if ngx.ctx.authenticated_consumer and conf.anonymous ~= "" then
```

Comment:
This was based on the logic in the official `jwt` plugin.  [See here](https://github.com/Kong/kong/blob/master/kong/plugins/jwt/handler.lua#L188)